.travis.yml

```diff
@@ -48,10 +48,11 @@ script:
   # Build ruby gem
   - gem build symengine.gemspec
   # Install ruby gem
-  - gem install symengine-0.1.0.gem --user --verbose -- -DSymEngine_DIR=$our_install_dir
+  - gem install symengine-0.1.0.gem --install-dir $HOME --verbose -- -DSymEngine_DIR=$our_install_dir
```

Comment:
Looks like the it's not able to find the .so file in OS X.
```
/Users/travis/gems/symengine-0.1.0/lib/symengine.rb:1:in `require': cannot load such file -- symengine/symengine (LoadError)
	from /Users/travis/gems/symengine-0.1.0/lib/symengine.rb:1:in `<top (required)>'
	from /Users/travis/gems/symengine-0.1.0/spec/spec_helper.rb:16:in `require_relative'
```
Are we missing something?